Library/Homebrew/test/utils/tty_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "utils"
+
+describe Tty do
+  subject { described_class }
```

Comment:
I don't _think_ this line is necessary. Haven't checked, but I think RSpec automatically sets `subject` like this when `described_class` is a module.